src/views/Epochs/EpochsView.tsx

```diff
@@ -20,21 +22,17 @@ type PanelProps = {
     }[]
 }
 
-const margins = {
-    left: 30,
-    right: 20,
-    top: 20,
-    bottom: 50
-}
-
 const panelSpacing = 4
 
 const EpochsView: FunctionComponent<Props> = ({data, width, height}) => {
     const {visibleTimeStartSeconds, visibleTimeEndSeconds } = useTimeRange()
 
+    const margins = useTimeseriesMargins(undefined)
+
     // Compute the per-panel pixel drawing area dimensions.
     const panelCount = 1
-    const { panelWidth, panelHeight } = usePanelDimensions(width, height, panelCount, panelSpacing, margins)
+    const toolbarWidth = DefaultToolbarWidth
+    const { panelWidth, panelHeight } = usePanelDimensions(width - toolbarWidth, height, panelCount, panelSpacing, margins)
```

Comment:
Upon reflection (looking at this & at the `PositionPdfPlotWidget` below), I think it might be easier to add a `toolbarWidth` parameter to the `usePanelDimensions` hook (perhaps an optional one defaulting to `DefaultToolbarWidth`?) so that you don't have to remember to subtract it out when using the hook.

Reply:
Or a `hideToolbar` boolean or something?